variables.tf

```diff
@@ -102,9 +98,28 @@ variable "vcn_name" {
 variable "drg_display_name" {
   description = "(Updatable) Name of Dynamic Routing Gateway. Does not have to be unique."
   type        = string
-  default     = null
+  default     = "drg"
+}
+
+variable "internet_gateway_display_name" {
+  description = "(Updatable) Name of Internet Gateway. Does not have to be unique."
+  type        = string
+  default     = "internet-gateway"
 }
 
+variable "nat_gateway_display_name" {
```

Comment:
Add a validation condition so the display name in the terraform.tfvars file is not an empty string: https://www.terraform.io/docs/language/values/variables.html#custom-validation-rules